src/App.jsx

```diff
@@ -1,86 +1,16 @@
 import React from 'react';
 import './App.css';
+import Story from './components/Article/Article';
+import Greetings from './components/Welcome/Welcome';
+import Head from './components/Header/Header';
```

Comment:
The improve should have correct names
```suggestion
import Article from './components/Article/Article';
import Welcome from './components/Welcome/Welcome';
import Header from './components/Header/Header';
```